pci/src/vfio.rs

```diff
@@ -396,6 +399,8 @@ pub(crate) struct VfioCommon {
     pub(crate) msi_interrupt_manager: Arc<dyn InterruptManager<GroupConfig = MsiIrqGroupConfig>>,
     pub(crate) legacy_interrupt_group: Option<Arc<dyn InterruptSourceGroup>>,
     pub(crate) vfio_wrapper: Arc<dyn Vfio>,
+    pub dwords: [u32; 1024],
```

Comment:
This is sloppy at best. This needs to go somewhere else, as its an internal detail to the Vfio implementation. This hack also required changes to the vfio_user.rs code